packages/dotcom-ui-header/src/header.scss

```diff
@@ -9,31 +9,11 @@
   display: block;
 }
 
-// z-indexes
-.o-header__mega {
-  @include nUiZIndexFor('meganav');
-}
-
-.o-header__drawer {
-  @include nUiZIndexFor('drawer');
-  display: block;
-  .o-header__drawer-search {
-    margin-bottom: 8px;
-  }
-}
-
-.o-header--sticky {
-  @include nUiZIndexFor('sticky-header');
-}
-
```

Comment:
**praise:** removing old CSS makes me so happy, especially when they are previous hacky overrides 🙌 Thank you!